### DISCUSSION TITLE
Add Twilio Sans Mono

### DISCUSSION THREAD

**Discussion by decipher3114** [ryanoasis/nerd-fonts#1295] [Ideas]
Its an amazing looking font with italics support.

https://github.com/twilio/twilio-sans-mono

**Comment by adityamwagh** [comment]
Yes please. I tried to patch it myself but it's not working.

**Reply by Finii** [reply]
Sorry to hear that.
Can you describe the problem?

**Reply by adityamwagh** [reply]
**UPDATE:** Running `fontforge -script font-patcher --complete <font_name>.ttf` fixed the issue for me.

I downloaded the font files from the link in the previous comment and ran `fontforge -script font-patcher <font_name>.ttf` and it generated the ttf files for all fonts I provided as an argument. Then I installed the fonts however it's not showing any icons in Windows Terminal.

## Settings Tab
![image](https://github.com/ryanoasis/nerd-fonts/assets/25822972/fbb9c685-2805-4c2e-a370-1ebf862ac8e7)

## Terminal Window
![image](https://github.com/ryanoasis/nerd-fonts/assets/25822972/84cccb15-4ea1-40a3-a95c-931ad2c313bb)

**Comment by adityamwagh** [comment]
@decipher3114 You can use this one meanwhile - https://github.com/adityamwagh/twilio-sans-mono-nerd-font
